common/djangoapps/student/models.py

```diff
@@ -445,6 +445,14 @@ class Meta(object):
     specialization = models.ManyToManyField(Specialization, blank=True)
     diploma_number = models.CharField(max_length=11, blank=True, null=True)
 
+    @property
+    def get_full_name(self):
```

Comment:
If you are going to use `property`, so do not use verbs as the function name `get_full_name`. It should be just `full_name`.

Reply:
Done